src/AutoRest.CSharp/Common/Generation/Writers/CodeWriterExtensions.cs

```diff
@@ -563,7 +572,7 @@ private static string GetConversion(CodeWriter writer, CSharpType from, CSharpTy
             return writer.Scope($"if ({propertyName} != null)");
         }
 
-        public static IDisposable WriteCommonMethodWithoutValidation(this CodeWriter writer, MethodSignature signature, FormattableString? returnDescription, bool isAsync, bool isPublicType, bool enableAttributes = false, IEnumerable<Attribute>? attributes = default)
+        public static IDisposable WriteCommonMethodWithoutValidation(this CodeWriter writer, MethodSignature signature, FormattableString? returnDescription, bool isAsync, bool isPublicType, IEnumerable<Attribute>? attributes = default)
```

Comment:
we do not need this flag to enable attributes. If there is any attribute passed in, we should always write them. Otherwise, they should not be passed in